cmd/geth/dbcmd.go

```diff
@@ -810,6 +820,61 @@ func dbDelete(ctx *cli.Context) error {
 	return nil
 }
 
+// dbDeleteTrieState deletes all trie state related key-value pairs from the database and the ancient state store.
+func dbDeleteTrieState(ctx *cli.Context) error {
+	if ctx.NArg() > 0 {
+		return fmt.Errorf("no arguments required")
+	}
+
+	stack, config := makeConfigNode(ctx)
+	defer stack.Close()
+
+	db := utils.MakeChainDatabase(ctx, stack, false, false)
+	defer db.Close()
+
+	var err error
+	if db.StateStore() != nil { // if separate trie db exists, delete all files in the db folder
```

Comment:
called rawdb.DeleteTrieState(db) directly and check whether the stateStore is nil in DeleteTrieState function.